packages/app/src/components/pages/Home/useSort.js

```diff
@@ -37,6 +37,10 @@ const useSort = (nomsToSort=[]) => {
         return sortConfig.direction === SORT_DIRECTION.DOWN ? -1 : 1
       }
 
+      if (aDate > bDate) {
```

Comment:
Do we need this? Seems like this is already covered on line 32.

Reply:
i think @t-keazirian only corrected spacing, this has nothing to do with our ticket, do you still want me to change it? I don't know what anything in the file does.

Reply:
Based on the diff, this whole block is new, not just a spacing change. We should be able to delete line 40-43.

Reply:
done